opentelemetry/proto/common/v1/common.proto

```diff
@@ -59,24 +59,35 @@ message KeyValueList {
   repeated KeyValue values = 1;
 }
 
-// KeyValue is a key-value pair that is used to store Span attributes, Link
+// Represents a key-value pair that is used to store Span attributes, Link
 // attributes, etc.
 message KeyValue {
+  // The key name of the pair.
   string key = 1;
+
+  // The value of the pair.
   AnyValue value = 2;
 }
 
 // InstrumentationScope is a message representing the instrumentation scope information
 // such as the fully qualified name and version. 
 message InstrumentationScope {
+  // A fully qualified name.
```

Comment:
I am not sure if we define that this is "fully qualified" name anywhere in the spec (and what that means). Do we?